lib/flatpak-remote.h

```diff
@@ -25,6 +25,15 @@
 #ifndef __FLATPAK_REMOTE_H__
 #define __FLATPAK_REMOTE_H__
 
+#ifdef FLATPAK_ENABLE_P2P
+typedef enum
```

Comment:
OK. I added the `#ifdef`s to avoid breaking public API later if they do have to change — but if you’re happy with that, I’m happy with removing that `#ifdef` mess. Do you want me to do that as an update to this PR, or as a follow-up PR?